scripts/scil_bundle_fixel_analysis.py

```diff
@@ -0,0 +1,291 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Analyze bundles at the fixel level using peaks and bundles (.trk). If the
+bundles files are names as {bundle_name}.trk, simply use the --in_bundles
+argument with --in_bundles_names. If it is not the case or you want other names
+to be saved, please use --in_bundles_names to provide bundles names IN THE SAME
+ORDER as the inputed bundles.
+
+The script produces various output:
+
+    - bundles_LUT.txt : array of (N) bundle names
+      Lookup table (LUT) to know the order of the bundles in the various
+      outputs. When an output contains results for all bundles, it is always
+      the last dimension of the np.ndarray and it follows the order of the
+      lookup table.
+
+    - fixel_density_maps.nii.gz : np.ndarray (x, y, z, 5, N)
```

Comment:
Please emphasize the files that are NOT 3D nifti files. The reader does not realize this.... In MI-Brain, the user will have to scroll through the 5th dimension for the 4D files and will not be able to interact with the 5D files, or will he?

Reply:
For the 5D files, the user will only see the first element of the 5th dimension as a 4D volume in MI-Brain.